mesheryctl/internal/cli/root/system/context/view.go

```diff
@@ -41,7 +44,7 @@ var viewContextCmd = &cobra.Command{
 			log.Printf("context \"%s\" doesn't exists, run the following to create:\n\nmesheryctl system context create %s", context, context)
 			return nil
 		}
-
+		log.Printf("\nContext: %s\n", context)
```

Comment:
I think as per our last discussion, we decided on `Current context:`.

Please refer to the mockup in the issue- #2829 #2830